n2v/inverter.py

```diff
@@ -127,16 +127,17 @@ def generate_mints_matrices(self):
         mints = psi4.core.MintsHelper( self.basis )
 
         #Overlap Matrices
-        self.S2 = mints.ao_overlap().np
+        self.S2 = np.array(mints.ao_overlap())
         A = mints.ao_overlap()
         A.power( -0.5, 1e-16 )
         self.A = A
         self.S3 = np.squeeze(mints.ao_3coverlap(self.basis,self.basis,self.pbs))
+        self.I = np.asarray(mints.ao_eri())
```

Comment:
Is this necessary for the calculation? Because the ao_eri matrices take quite a lot memory.

Reply:
Is it possible to still use the old fashion psi4.jk function? I feel like that will save quite some memory since eri is highly symmetric.

Reply:
Done. Replaced all dependence on `I` with `build_jk` as before.